sdk/core/azure-core-serializer-json-gson/src/main/java/com/azure/core/serializer/json/gson/GsonJsonNull.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.serializer.json.gson;
+
+import com.azure.core.util.serializer.JsonNull;
+
+import java.util.Objects;
+
+/**
+ * GSON specific implementation of {@link JsonNull}.
+ */
+public final class GsonJsonNull implements JsonNull {
+    private final com.google.gson.JsonNull jsonNull;
```

Comment:
thoughts on `INSTANCE` public final static property of type `GsonJsonNull`?

Reply:
Added `INSTANCE`.